File: Python_Design-Patterns/Taller_Adapter/Bus.py
from MedioTransporte import *

class Bus(MedioTransporte):

    __costo = 0
    __horario = ''

    def __init__(self):
        print("Creando Bus")

    def getCosto(self):
        return __costo 
    
    def setCosto(self, costo):
        __costo = costo

    def getHorario(self):
        return __horario

    def setHorario(self, horario):
        __horario = horario

File: Java_Design-Patterns/src/talleradapter/Bus.java
package TallerAdapter;

public class Bus extends MedioTransporte {

    private double costo;
    private String horario;

    public Bus() {
        super();
        System.out.println("Creando Bus");
    }

    @Override
    public double getCosto() {
        return costo;
    }

    @Override
    public void setCosto(double costo) {
        this.costo = costo;
    }

    @Override
    public String getHorario() {
        return horario;
    }

    @Override
    public void setHorario(String horario) {
        this.horario = horario;
    }
}
